### DISCUSSION TITLE
Source code zip differs from assets

### DISCUSSION THREAD

**Discussion by Frostice2013** [community/community#86599] [New to GitHub] [answered]
hi, i have a repository and for some reason the source code zip actually differs from the assets, how does this happen? ive tried alot of things so far but none of them work

**Comment by queenofcorgis** [comment]
👋🏼 @Frostice2013, 

Would you mind sharing with us how it differs and what you've tried so far so we can better assist?

**Comment by Frostice2013** [comment, marked as answer]
op, apparently just had to change release tags for changes to actually happen on the source code zip, thanks queenofcorgis for the help!

**Reply by queenofcorgis** [reply]
@Frostice2013 awesome - thanks for coming back and sharing the solution!
